source/mainnet/net/installation/previous-node-downloads.rst

```diff
@@ -0,0 +1,198 @@
+
+:orphan:
+
+.. _previous-downloads:
+
+=======================
+Previous node downloads
+=======================
+
+This topic contains downloads for previous node versions if you need to upgrade a node over multiple versions.
+
+Windows - Mainnet and Testnet
+=============================
+
+**Please be aware that you should backup your configuration, as the installer will overwrite the current configuration with a standard configuration.**
+
+`Windows 4.2.2 <https://distribution.concordium.software/windows/Signed/Node-4.2.2-0.msi>`_
+
+`Windows 4.2.1 <https://distribution.concordium.software/windows/Signed/Node-4.2.1-0.msi>`_
+
+`Windows 4.1.1 <https://distribution.concordium.software/windows/Signed/Node-4.1.1.msi>`_ (Note: new genesis)
+
+`Windows 4.1.0 <https://distribution.concordium.software/windows/Signed/Node-4.1.0-0.msi>`_
+
+`Windows 4.0.11 <https://distribution.concordium.software/windows/Signed/Node-4.0.11-0.msi>`_
+
+`Windows 4.0.7 <https://distribution.concordium.software/windows/Signed/Node-4.0.7.msi>`_
```

Comment:
I don't think we released these for test & main net. Only stagenet.

Reply:
4.0.11 was released but only for testnet.